tools/releasing/collect_license_files.sh

```diff
@@ -57,4 +57,9 @@ LICENSES="${DST}/licenses"
 [ -f "${LICENSES}" ] && rm -r "${LICENSES}"
 find "${TMP}" -name "licenses" -type d -exec cp -r -- "{}" "${DST}" \;
 
+# Search the binary distribution directory and collect those license files that
+# not bundled in any jars.
+find "${SRC}/flink-dist/target" -name "LICENSE.*" -type f \
+-path "*/flink-*-bin/*" ! -path "*/licenses/*" -exec cp -- "{}" "${DST}/licenses" \;
```

Comment:
- `-path "*/flink-*-bin/*"` should not be necessary with the above suggestion.

- It may be worthwhile to explicitly exclude `${TMP}/licenses` and `${DST}/licenses`; this script should continue to work even if the python licenses are moved into to `opt/python/licenses`.

- maybe to a move instead so we have a single point for licenses.

Reply:
+1 for the 1st and 2nd comments. For 3rd comments I'm not sure it is a good idea, because "move" is not a idempotent and safe operation. If someone run this script multiple times the results may not be what they expected. If someone run this script in other directories, there may be unexpected consequences, such as the strange disappearance of many files.